lib/policy_machine/policy_element.rb

```diff
@@ -264,6 +293,23 @@ def allowed_assignee_classes
     end
   end
 
+  # A prohibition in a policy machine.
+  class Prohibition < PolicyElement
+    def self.new(operation)
+      negation = "~#{operation}"
+      case operation
+      when PM::Operation
+        PM::Operation.find_or_create(negation,operation.policy_machine_uuid, operation.pm_storage_adapter, {}, true)
+      when Symbol
+        negation.to_sym
```

Comment:
https://github.com/mdsol/the_policy_machine/pull/27/files#diff-cd736b91a0f9140de3bb79b73ec78b70R299 <code>negation</code> is a string